.github/workflows/draft-release.yml

```diff
@@ -112,16 +109,17 @@ jobs:
           lifecycle_path=$(ls artifact-windows/lifecycle-*windows.x86-64.tgz)
           echo "ARTIFACT_WINDOWS_PATH=$PWD/$lifecycle_path" >> $GITHUB_ENV
       - name: Create Pre Release
-        if: contains(env.LIFECYCLE_VERSION, 'rc')
+        if: contains(env.LIFECYCLE_VERSION, 'rc') # e.g., 0.99.0-rc.1
         id: create_pre_release
-        uses: actions/create-release@latest
+        uses: softprops/action-gh-release@v1
```

Comment:
Have we used this action elsewhere? Do we need to vet any new actions?

Reply:
It's referred to by https://github.com/actions/create-release and used in `pack`, but I discovered some issues in how we were using it (see https://github.com/buildpacks/pack/issues/1213) and from [this thread](https://github.com/softprops/action-gh-release/issues/107) I decided to switch to using `gh` cli. Now commits coming soon...